### PR TITLE
fix(textarea): use correct styling and stories

### PR DESCRIPTION
**Pull request by notrab**
No changeset, previous one wasn't merged yet.

### CODE REVIEW

File: packages/namekit-react/src/components/ENSTextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import cc from "classcat";
+
+import { TextArea, TextAreaProps } from "./TextArea";
+
+export const ENSTextarea: React.FC<TextAreaProps> = (props) => {
```
Comment (lightwalker-eth) [inline packages/namekit-react/src/components/ENSTextArea.tsx:6]: @notrab suggest we also rename this to `ENSTextArea` with an uppercase `A` and also update related items in Storybook, ex:

CleanShot 2024-07-03 at 10 13 37

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@notrab Thanks for updates. All looks good except for a capitalization suggestion.

File: packages/namekit-react/src/components/ENSTextArea.tsx (hunk 1 above)
Comment (notrab) [reply packages/namekit-react/src/components/ENSTextArea.tsx]: Good catch, fixed

**Review by lightwalker-eth — APPROVED** [review]
@notrab Thanks, looks good :+1:

**Pull request merged**